### PR TITLE
sct_events: cleanup of log levels

### PR DESCRIPTION
**Pull request by fruch**
• now each level has it's own log
• new summary.log with counts of each level
• summery is added to the email reports

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/sct_events.py [hunk 1]
```diff
@@ -587,29 +588,50 @@ def run(self):
...
+class EventsFileLogger(Process):  # pylint: disable=too-many-instance-attributes
     def __init__(self, log_dir):
         super(EventsFileLogger, self).__init__()
         self._test_pid = os.getpid()
         self.event_log_base_dir = os.path.join(log_dir, 'events_log')
         self.events_filename = os.path.join(self.event_log_base_dir, 'events.log')
         self.critical_events_filename = os.path.join(self.event_log_base_dir, 'critical.log')
-        with open(self.critical_events_filename, 'a'):
-            pass
+        self.error_events_filename = os.path.join(self.event_log_base_dir, 'error.log')
```
Comment (bentsi) [inline sdcm/sct_events.py]: We are in Python 3! Use Path 
```
In [1]: from pathlib import Path                                                                                                                                                                                   

In [2]: p = Path("/tmp/file")                                                                                                                                                                                      

In [3]: p.touch()   
```

File: sdcm/results_longevity.html [hunk 2]
```diff
@@ -106,6 +106,13 @@ <h3>
             {% endfor %}
         </div>
     {% endif %}
+    <div>
```
Comment (bentsi) [inline sdcm/results_longevity.html]: Add heading
Events summary

File: sdcm/sct_events.py [hunk 3]
```diff
@@ -587,29 +588,50 @@ def run(self):
...
-                if (message_data.severity == Severity.CRITICAL or message_data.severity == Severity.ERROR) \
-                        and not event_type == 'ClusterHealthValidatorEvent':
-                    with open(self.critical_events_filename, 'a+') as critical_file:
-                        critical_file.write(msg + '\n')
+                # update each level log file
+                events_filename = self.level_to_file_mapping[message_data.severity]
+                with open(events_filename, 'a+') as events_level_file:
+                    events_level_file.write(msg + '\n')
+
+                # update the summery file
```
Comment (bentsi) [inline sdcm/sct_events.py]: summary

**Comment by aleksbykov** [comment]
@fruch do we decide that test failed if files critical.log and error.log are not empty? 
if both required to empty, need to update the common.py:get_testrun_status function

**Comment by fruch** [comment]
@aleksbykov no we want to fail test only on critical issues for now

**Pull request merged**